packages/vaadin-combo-box/test/item-renderer.test.js

```diff
@@ -1,100 +1,78 @@
 import { expect } from '@esm-bundle/chai';
 import sinon from 'sinon';
 import { fixtureSync } from '@vaadin/testing-helpers';
+import '@vaadin/vaadin-template-renderer';
 import './not-animated-styles.js';
 import '../vaadin-combo-box.js';
 
 describe('form field', () => {
   let comboBox;
 
-  describe('without template', () => {
-    let items;
-
-    beforeEach(() => {
-      comboBox = fixtureSync(`
-        <vaadin-combo-box
-          item-label-path="name"
-          item-value-path="symbol">
-        </vaadin-combo-box>
-      `);
-      items = ['foo', 'bar', 'baz'];
-      comboBox.items = items;
-    });
-
-    afterEach(() => {
-      comboBox.opened = false;
-    });
-
-    it('should use renderer when it is defined', () => {
-      comboBox.renderer = (root, comboBox, model) => {
-        const textNode = document.createTextNode(`${model.item} ${model.index}`);
-        root.appendChild(textNode);
-      };
-      comboBox.opened = true;
+  let items;
+
+  beforeEach(() => {
+    comboBox = fixtureSync(`
+      <vaadin-combo-box
+        item-label-path="name"
+        item-value-path="symbol">
+      </vaadin-combo-box>
+    `);
+    items = ['foo', 'bar', 'baz'];
+    comboBox.items = items;
+  });
 
-      const item = comboBox.$.overlay._selector.querySelector('vaadin-combo-box-item');
-      expect(item.$.content.textContent.trim()).to.equal('foo 0');
-    });
+  afterEach(() => {
+    comboBox.opened = false;
+  });
 
-    it('renderer should receive root, comboBox and model', (done) => {
-      let isDone = false;
+  it('should use renderer when it is defined', () => {
+    comboBox.renderer = (root, comboBox, model) => {
+      const textNode = document.createTextNode(`${model.item} ${model.index}`);
+      root.appendChild(textNode);
+    };
+    comboBox.opened = true;
 
-      comboBox.renderer = (root, comboBox, model) => {
-        expect(root.getAttribute('part')).to.equal('content');
-        expect(items.indexOf(model.item)).to.not.equal(-1);
-        expect(comboBox).to.eql(comboBox);
+    const item = comboBox.$.overlay._selector.querySelector('vaadin-combo-box-item');
+    expect(item.$.content.textContent.trim()).to.equal('foo 0');
+  });
 
-        if (!isDone) {
-          isDone = true;
-          done();
-        }
-      };
+  it('renderer should receive root, comboBox and model', (done) => {
```

Comment:
We probably need another test for `selected` and `focused` properties on the `model`, do we have one?

Reply:
Sure, thanks! I will add them.

Reply:
Added a few more tests.